adafruit_thermal_printer/thermal_printer.py

```diff
@@ -117,13 +123,17 @@ class _PrintModeBit:
         # which by design only implements get, set, init.  As a result workaround
         # this pylint issue by disabling the warning.
         # pylint: disable=too-few-public-methods
-        def __init__(self, mask):
+        def __init__(self, mask: int) -> None:
             self._mask = mask
 
-        def __get__(self, obj, objtype):
+        def __get__(
+            self,
+            obj: Optional["ThermalPrinter"],
+            objtype: Optional[Type["ThermalPrinter"]],
```

Comment:
`objtype` shouldn't really ever be `None` as far as a I know.  The only reason it's been annotated as `Optional` in the past is because it's default value was `None`, but truly it should always be the type of `ThermalPrinter`.